androidNode/src/androidMain/kotlin/network/bisq/mobile/android/node/service/offers/NodeOffersServiceFacade.kt

```diff
@@ -324,9 +404,10 @@ class NodeOffersServiceFacade(
                 channel.market.baseCurrencyName,
                 channel.market.quoteCurrencyName,
             )
+            val count = channel.chatMessages.count { it.hasBisqEasyOffer() && isValidOfferbookMessage(it) }
```

Comment:
As we always check for both `it.hasBisqEasyOffer() && isValidOfferbookMessage(it)` we should maybe extract it in a method